<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HgCopyDialog</class>
 <widget class="QDialog" name="HgCopyDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>120</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mercurial Copy</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="textLabel1">
     <property name="text">
      <string>Source:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="sourceEdit">
     <property name="toolTip">
      <string>Shows the name of the source</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Source name&lt;/b&gt;
&lt;p&gt;This field shows the name of the source.&lt;/p&gt;</string>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="textLabel2">
     <property name="text">
      <string>Target:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="E5PathPicker" name="targetPicker" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="toolTip">
      <string>Enter the target name</string>
     </property>
     <property name="whatsThis">
      <string>&lt;b&gt;Target name&lt;/b&gt;
&lt;p&gt;Enter the new name in this field. The target must be the new name or an absolute path.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QCheckBox" name="forceCheckBox">
     <property name="toolTip">
      <string>Select to force the operation</string>
     </property>
     <property name="text">
      <string>Enforce operation</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>E5PathPicker</class>
   <extends>QWidget</extends>
   <header>E5Gui/E5PathPicker.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>sourceEdit</tabstop>
  <tabstop>targetPicker</tabstop>
  <tabstop>forceCheckBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>HgCopyDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>32</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>32</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>HgCopyDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>101</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>101</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
